design/authenticators/authn_k8s_error_solution.md

```diff
@@ -0,0 +1,54 @@
+**Note**: This design document has been ported from the original document
+[here](https://github.com/cyberark/conjur/issues/524).
+
+# Background
+The need to write this solution design arises from the problem that was raised as part of secret provider phase II implementation.
```

Comment:
please link to the solution doc

Reply:
please, send me a link to the solution doc and I will add it

Reply:
https://github.com/cyberark/secrets-provider-for-k8s/blob/master/design/milestone_1_2_design_doc.md